src/components/CoinSyncEmptyState/index.jsx

```
@@ -0,0 +1,48 @@
+    subtitle: "We'll check all supported coins for balances",
+  },
+  completed: {
+    title: 'All caught up!',
+    subtitle: 'No additional coins with balance found',
+  },
+};
+
+const CoinSyncEmptyState = ({status, hasCoinsWithBalance}) => {
+  const config = useMemo(() => {
+    if (status === 'fetching') return EMPTY_STATE_CONFIG.fetching;
+    if (status === 'creating_wallets') return EMPTY_STATE_CONFIG.creating_wallets;
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)